packages/jellyfish-api-core/__tests__/category/rawtx/decodeScript.test.ts

```diff
@@ -0,0 +1,136 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import { wallet } from '../../../src'
+
+describe('Decodescript()', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await container.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should decode empty script', async () => {
+    const address = ''
+    const decode = await testing.rpc.rawtx.decodeScript(address)
```

Comment:
I'm surprised of this scenario, how come?

Reply:
My understanding is such that in string inputs for `decodeScript()` would need to be greater than 0 for any decoding to take place. However, if the string is empty, it will return an empty value. 

`e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` in the results refer to a (hashed) zero-length file.

In reference to `ain`:
```
    UniValue r(UniValue::VOBJ);
    CScript script;
    if (request.params[0].get_str().size() > 0){
        std::vector<unsigned char> scriptData(ParseHexV(request.params[0], "argument"));
        script = CScript(scriptData.begin(), scriptData.end());
    } else {
        // Empty scripts are valid
    }
    ScriptPubKeyToUniv(script, r, /* fIncludeHex */ false);
```
Please let me know if that clarifies your question or if I had misunderstood your question.